src/utils/index.ts

```diff
@@ -105,3 +106,15 @@ export const shortAddress = (text: string, size = 24) => {
 
     return `${start}...${end}`
 }
+
+export const validateAddress = (connectionString): boolean => {
+    if (!connectionString || typeof connectionString !== 'string') {
+        return false
+    }
+
+    if (connectionString.toUpperCase().startsWith('COM')) {
+        return /^COM\d+$/i.test(connectionString)
+    }
+
+    return IP_ADDRESS_REGEX.test(connectionString)
```

Comment:
Can we also add support for UVC cameras? It's just an index from 0 to 9. I know that the backend will also have to support it but that I can add later